server/routes/api/landing_page.py

```diff
@@ -518,20 +554,27 @@ def populate_category_data(category):
             # If there is no data for a category in overview page, need to
             # "borrow" it from the category page.
             if not has_data(data):
-                populate_category_data(category)
-                total_charts = 0
-                spec_and_stat[OVERVIEW][category] = []
-                for topic in spec_and_stat[category]:
-                    spec_and_stat[OVERVIEW][category].extend(
-                        spec_and_stat[category][topic])
-                    total_charts += len(spec_and_stat[category][topic])
-                    if total_charts > MAX_OVERVIEW_CHART_GROUP:
-                        break
+                populate_additional_category_data(category)
 
     # Get chart category name translations
     categories = {}
-    for category in list(spec_and_stat.keys()) + list(spec_and_stat[OVERVIEW]):
-        categories[category] = gettext(f'CHART_TITLE-CHART_CATEGORY-{category}')
+    config_categories = set()
+    ordered_categories = []
+    for conf in current_app.config['CHART_CONFIG']:
+        config_categories.add(conf['category'])
+        ordered_categories.append(conf['category'])
+    for category in list(spec_and_stat.keys()) + list(
+            spec_and_stat[OVERVIEW]
+    ) + raw_page_data["validCategories"][dcid]['category']:
```

Comment:
Better readability to make a variable to hold this concatenated list.

Reply:
Done